src/pages/payment-methods/_expanded-list.tsx

```diff
@@ -1,20 +1,26 @@
-import React from 'react'
-import PropTypes from 'prop-types'
+import React, { ReactElement } from 'react'
 import styled, { css } from 'styled-components'
 import { Button } from 'components/form/'
 import { Text } from 'components/elements'
 import { localize } from 'components/localization'
-// import { getCryptoDecimals } from 'common/utility'
-// SVG
 import Chevron from 'images/svg/custom/chevron-thick.svg'
 import PDF from 'images/svg/regulatory/pdf-icon-black.svg'
 
+type StyledChevronType = {
+    expanded: boolean
+}
+type TrType = {
+    is_expanded?: boolean
+}
+type DepositType = {
+    is_fiat_onramp: boolean
+}
+
```

Comment:
Can we cluster this in one into a reusable type so we don't need to declare 3 types something like:

```
 type SomeType = {
   is_expanded?: boolean
   is_fiat_onramp?: boolean
   }
   ```